docs/source/get_started/access_info/access_info_qss.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>Link for "this guide" leads to 404 like in the css version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>